bin/run.rb

```diff
@@ -0,0 +1,4 @@
+require_relative '../lib/game.rb'
+
+game = Game.new 
+game.welcome
```

Comment:
In general, try to format your files so that there is a newline at the end of each one. You can set your editor to do this automatically.

Reply:
Added a new line to each file
 76d2d7cd28cf075d602df76e03aa25ee10becbcd